index.html

```diff
@@ -387,7 +387,7 @@ <h3 class="about__connect fz-16 fw-500 lh-24 clr-p-400">
                                 Languages
                             </h3>
                             <img class="about__lang__top__img"
-                                src="images/arrow.png" alt="arrow icon" />
+                                src="images/arrow.png" alt="down arrow" />
```

Comment:
- [ ] Please use `alt=""` for all the decorative images or icons because they are just for decoration not the part of the content.